### DISCUSSION TITLE
Search Filter M2A (behavior dropdown)

### DISCUSSION THREAD

**Discussion by gloribus** [directus/directus#21720] [Feature Requests]
### Summary

Ability to use values from a related table (dictionary) when using a filter

### Motivation

It is often necessary to have reference information in a separate table, but the current release requires entering the name manually. This creates additional complexity

### Detailed Design

If you select a type as a dropdown option, the system will allow you to select a value from the list. Need to add automatic loading from the linked table

### Must Have:
- A A new type that will allow filtering to load data from a related table

### Should Have:
- B Search inside a loaded list

### Drawbacks

Time to implement

### Alternatives

Every time there are changes in the dictionary table, make changes to the JSON options of the dropdown list

### Adoption Strategy

This is an additional useful feature that does not affect current users

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
